spec/services/api/create_params_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Api::CreateParams do
+  describe "#permissions" do
+    it "allows scalar model attributes" do
+      pl_permissions = Api::CreateParams.new(Api::Model.new('plant_line'), {}).permissions
+      # FIXME plant_variety_name - shouldn't it be removed?
```

Comment:
PL belongs to PV and also has `plant_variety_name` attribute. Isn't that
redundant?

2015-06-26 12:18 GMT+02:00 Tomasz Gubała <notifications@github.com>:

> In spec/services/api/create_params_spec.rb
> <https://github.com/eSpectrum-IT/brassica/pull/328#discussion_r33344083>:
>
> > @@ -0,0 +1,53 @@
> > +require 'rails_helper'
> > +
> > +RSpec.describe Api::CreateParams do
> > +  describe "#permissions" do
> > +    it "allows scalar model attributes" do
> > +      pl_permissions = Api::CreateParams.new(Api::Model.new('plant_line'), {}).permissions
> > +      # FIXME plant_variety_name - shouldn't it be removed?
>
> What do you mean by that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/328/files#r33344083>.
>



-- 
Tomasz Szymczyszyn